### DISCUSSION TITLE
Probably a stoplight question but.....how do I create a "page" that just displays a markdown file?

### DISCUSSION THREAD

**Discussion by bclements** [danielgtaylor/huma#373] [Q&A]
I need to add some sumplementary information for my api in the stoplight docs. I read somewhere that stoplight supports having a custom link that points to a markdown file but can't figure out how to do it. 

@danielgtaylor you can close this if I need to go to the stoplight forums.

**Comment by danielgtaylor** [comment]
You might be talking about the `externalDocs` OpenAPI feature, which Stoplight may render. For example, see https://github.com/danielgtaylor/huma/blob/main/openapi.go#L885 where you can set it on an operation to link to additional documentation.
